libs/ui/elements/src/lib/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -0,0 +1,12 @@
+<h1 mat-dialog-title>{{ data.title }}</h1>
+<div mat-dialog-content>{{ data.message }}</div>
+<div mat-dialog-actions>
+  <gn-ui-button (buttonClick)="onCancel()">{{ data.cancelText }}</gn-ui-button>
+  <gn-ui-button
+    (buttonClick)="onConfirm()"
+    cdkFocusInitial
```

Comment:
Oh right, that was about accessibility, but the last focused element is the action menu button, which has been removed when the modal was opened... I guess we'll just let the focus go to the next element in focus order.

Reply:
Hm in fact it's how we preselect the confirm button when opening the dialog, otherwise it's the cancel button that is selected first. What's the most strange is that the directive actually works, even with the warning... Is it that bad to keep the warning?

Reply:
I agree it's important to keep it; it's really not that bad to have a warning in the console, it can be considered technical debt :)